vmdk_plugin/constants.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+const (
+	// vDVS drivers.
+	photonDriver  = "photon"
+	vmdkDriver    = "vmdk"
```

Comment:
Nit: Let's add an ending comment to mark it as deprecated:

   vmdkDriver    = "vmdk"   // deprecated